src/components/ArticlePage.jsx

```diff
@@ -0,0 +1,37 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import fetchArticles from "../../utils/fetchArticles";
+import "../styles/ArticlePage.css";
+
+export default function ArticlePage() {
+  const { article_id } = useParams();
+  const [article, setArticle] = useState({});
+
+  useEffect(() => {
+    fetchArticles(article_id).then((data) => {
+      setArticle({ ...data.article });
+    });
+  });
```

Comment:
I can't see any `isLoading` pattern here, at the moment the content doesn't render until the api call has finished. In the mean time (if I slow the connection down to low throttle manually) it's just blank.

Please implement an isLoading pattern for a better user experience, to let them know that the article is loading.

Also think about what happens if the article api call fails? How do you want to display an error to the user?